src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -0,0 +1,136 @@
+import type { Router as RemixRouter } from '@remix-run/router';
+import { screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import { CREATE_BLUEPRINT } from '../../../../../constants';
+import {
+  blueprintRequest,
+  clickBack,
+  clickNext,
+  enterBlueprintName,
+  getNextButton,
+  interceptBlueprintRequest,
+  openAndDismissSaveAndBuildModal,
+  verifyCancelButton,
+} from '../../wizardTestUtils';
+import {
+  clickRegisterLater,
+  goToRegistrationStep,
+  renderCreateMode,
+} from '../../wizardTestUtils';
+
+let router: RemixRouter | undefined = undefined;
+
+const goToUsersStep = async () => {
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+  await clickNext();
+};
+
+const goToReviewStep = async () => {
+  await clickNext(); // Timezone
+  await clickNext(); // Locale
+  await clickNext(); // Hostname
+  await clickNext(); // Kernel
+  await clickNext(); // First boot
+  await clickNext(); // Details
+  await enterBlueprintName();
+  await clickNext(); // Review
+};
+
+const addValidUser = async () => {
+  const user = userEvent.setup();
+  const addUser = await screen.findByRole('button', { name: /add a user/i });
+  expect(addUser).toBeEnabled();
+  await waitFor(() => user.click(addUser));
+  const enterUserName = screen.getByRole('textbox', {
+    name: /blueprint user name/i,
+  });
+  const nextButton = await getNextButton();
+  await waitFor(() => user.type(enterUserName, 'best'));
+  await waitFor(() => expect(enterUserName).toHaveValue('best'));
+  await waitFor(() => expect(nextButton).toBeEnabled());
+};
+
+describe('Step Users', () => {
+  beforeEach(async () => {
+    vi.clearAllMocks();
+    router = undefined;
+  });
+
+  test('without adding user loads First Boot', async () => {
```

Comment:
I think this test needs to be renamed to reflect what it's doing - it does add a user.

Reply:
this test when user get to the 'add a user' screen, he press Next and actually doesnt add a new user. `
just make sure, that user can skip this step and doesnt add a new user

Reply:
Ah I see, you're right. But I think the name of the test still needs some work, this says "without adding user loads First Boot" but the next step is actually timezone.